crates/revmjs/src/lib.rs

```diff
@@ -90,7 +91,8 @@ impl EVM {
     /****** ALL CFG ENV SETTERS ********/
 
     pub fn cfg_chain_id(&mut self, gas_limit: BN) {
-        self.revm.env.cfg.chain_id = gas_limit.try_into().unwrap();
+        self.revm.env.cfg.chain_id =
+            U256::from_limbs(primitive_types::U256::try_from(gas_limit).unwrap().0);
```

Comment:
All of these should turn int `gas_limit.into()`. But I think I need to add `BN` support to `ruint` first.

Reply:
That would be the best way.

Reply:
done